django/db/migrations/questioner.py

```diff
@@ -87,20 +87,26 @@ def ask_unique_callable_default_addition(self, field_name, model_name):
 
 
 class InteractiveMigrationQuestioner(MigrationQuestioner):
+    def __init__(self, defaults=None, specified_apps=None, dry_run=None, prompt_destination=sys.stdout):
+        super().__init__(defaults=defaults, specified_apps=specified_apps, dry_run=dry_run)
+        self.prompt_destination = prompt_destination
 
     def _boolean_input(self, question, default=None):
-        result = input("%s " % question)
+        self.prompt_destination.write("%s " % question)
+        result = input()
         if not result and default is not None:
             return default
         while not result or result[0].lower() not in "yn":
-            result = input("Please answer yes or no: ")
+            self.prompt_destination.write('Please answer yes or no: ')
+            result = input()
         return result[0].lower() == "y"
 
     def _choice_input(self, question, choices):
-        print(question)
+        self.prompt_destination.write(question + os.linesep)
```

Comment:
Do we need to use `os.linesep`?

Reply:
I was trying to remedy test failures on Windows, but I think I figured out how to just add `\n` in the calls replacing `print()`.